easybuild/easyconfigs/t/Trimmomatic/Trimmomatic-0.32-Java-1.7.0_21.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'Tarball'
+
+name = 'Trimmomatic'
+version = '0.32'
+
+homepage = 'http://www.usadellab.org/cms/?page=trimmomatic'
+description = """Trimmomatic performs a variety of useful trimming tasks for illumina 
+ paired-end and single ended data.The selection of trimming steps and their associated 
+ parameters are supplied on the command line. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/']
+sources = [SOURCE_ZIP]
+
+java = 'Java'
+javaver = '1.7.0_21'
```

Comment:
LGTM; only a hint, not a blocker: java 1.7.0_80 is a bit more preferred (if, for common deps)